environments/india/inventory.ini

```diff
@@ -61,8 +61,8 @@ elasticsearch_data=false
 
 [formplayer3]
 10.203.10.32 hostname=formplayer3-india ufw_private_interface=ens5 ansible_python_interpreter=/usr/bin/python3 ec2_instance_id=i-059f6df8ba2514bdd root_encryption_mode=aws
-[pillow4]
-10.203.10.164 hostname=pillow4-india ufw_private_interface=ens5 ansible_python_interpreter=/usr/bin/python3 ec2_instance_id=i-094e24ac7fd45b4ca
+[pillow5]
+10.203.10.53 hostname=pillow5-india ufw_private_interface=ens5 ansible_python_interpreter=/usr/bin/python3 ec2_instance_id=i-0aaae69fbfdf3ff59 root_encryption_mode=aws
```

Comment:
@pvisweswar do we know what caused this difference?

Reply:
I removed pillow4(old node)  entry from app-process.yml file then the PR checks are not passed and pushing an error as below, So I removed from inventory files then It is successfull. 

> `AssertionError: pillowtop machine 10.203.10.164 not in commcare-environments/india/app-processes.yml`

Reply:
Ah I missed that we set encryption to yes in the terraform file (Thanks for pointing that out Vishwa).